brands/shop/newagent.json

```diff
@@ -0,0 +1,32 @@
+{
+  "shop/newagent|Martin's": {
```

Comment:
the `newagent` here in the id is causing it to generate a new file called `newagent.json`

Reply:
I see the new file was created, but can't find any mention of 'newagent' in the shop/newsagent file... should I delete the newagent.json file and run npm run build again?

Reply:
yes delete the file, and run `npm run build` again.